website/content/docs/connect/gateways/api-gateway/tech-specs.mdx

```diff
@@ -0,0 +1,157 @@
+---
+layout: docs
+page_title: API gateway for Kubernetes technical specifications
+description: >-
+  Learn about the requirements for installing and using the Consul API gateway for Kubernetes, including required ports, component version minimums, Consul Enterprise limitations, and compatible k8s cloud environments.
+---
+
+# API gateway for Kubernetes technical specifications
+
+This topic describes the requirements and technical specifications associated with using Consul API gateway.
+
+## Datacenter requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API gateway:
+
+- HashiCorp Consul Helm chart v1.2.0 and later
+
+## TCP port requirements
+
+The following table describes the TCP port requirements for each component of the API gateway.
+
+| Port | Description | Component |
+| ---- | ----------- | --------- |
+| 20000 | Kubernetes readiness probe | Gateway instance pod |
+| Configurable | Port for scraping Prometheus metrics. Disabled by default. | Gateway controller pod |
+
+## OpenShift requirements
+
+You can deploy API gateways to Kubernetes clusters managed by Red Hat OpenShift, which is a security-conscious, opinionated wrapper for Kubernetes. To enable OpenShift support, add the following parameters to your Consul values file and apply the configuration:
+
+```yaml
+  openshift:
+    enabled: true
+ ```
+
+Refer to the following topics for additional information:
+
+- [Install Consul on OpenShift clusters with Helm](/consul/docs/k8s/installation/install#install-consul-on-openshift-clusters)
+- [Install Consul on OpenShift clusters with the `consul-k8s` CLI](/consul/docs/k8s/installation/install-cli#install-consul-on-openshift-clusters)
+
+### Security context constraints
+
+OpenShift requires a security context constraint (SCC) configuration, which restricts pods to specific groups. You can create a custom SCC or use one of the default constraints. Refer to the [OpenShift documentation](https://docs.openshift.com/container-platform/4.13/authentication/managing-security-context-constraints.html) for additional information. 
+
+By default, the SCC is set to `restricted-v2` for the `managedGatewayClass` that Consul automatically creates. The `restricted-v2` SCC is one of OpenShifts default SCCs, but you can specify a different SCC in the `openshiftSccName` parameter:
+
+```yaml
+connectInject:
+  apiGateway:
+    managedGatewayClass:
+      openshiftSccName: "restricted-v2"
+```
+
+### Privileged container ports
+ 
+Containers cannot use privileged ports when OpenShift is enabled. Privileged ports are 1 through 1024, and serving applications from that range is a security risk.
```

Comment:
ports are 1-1023 inclusive, or 1-1024 exclusive. My brain assumes "inclusive" when reading the above, but I'm not sure if that's just me